### PR TITLE
Merge v2.5.0

### PR DESCRIPTION
**Pull request by cooper-lzy**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -29,22 +29,14 @@ jobs:
           pip3 install -r ./requirements.txt
 
       - name: Git Config
-        run: git config user.name jerry.liang && git config user.email ${{secrets.GIT_EMAIL}}
+        run: git config user.name whitewum && git config user.email min.wu@vesoft.com
 
-      - name: mike list delete
+      - name: Mike Deploy
```
Comment (whitewum) [inline .github/workflows/deploy.yml:34]: leave this file unchanged

Comment (whitewum) [reply .github/workflows/deploy.yml:34]: as master 's version

File: mkdocs.yml [hunk 2]
```diff
@@ -20,8 +20,8 @@ theme:
     logo: 'https://cloud-cdn.nebula-graph.com.cn/vesoft-nebula-logo-white.png'
     favicon: 'assets/images/favicon.ico'
     palette:
-      primary: 'teal'
-      accent: 'light green'
+      primary: 'cyan'
```
Comment (whitewum) [inline mkdocs.yml:23]: the master is right.

File: mkdocs.yml [hunk 3]
```diff
@@ -20,8 +20,8 @@ theme:
...
+      accent: 'lime'
     features:
       - navigation.top
     language: 'en'
```
Comment (whitewum) [inline mkdocs.yml:27]: make relase as master

**Review by whitewum — COMMENTED** [review]
don't delete header.html

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -29,22 +29,14 @@ jobs:
...
 
       - name: Git Config
-        run: git config user.name jerry.liang && git config user.email ${{secrets.GIT_EMAIL}}
+        run: git config user.name whitewum && git config user.email min.wu@vesoft.com
 
       - name: mike list delete
         run: |
-          #mike delete 2.0.0 -p
+          mike deploy 2.5.0 -p --rebase
+          mike set-default 2.5.0 -p --rebase
```
Comment (whitewum) [inline .github/workflows/deploy.yml:37]: wrong

File: docs-2.0/1.introduction/3.vid.md [hunk 5]
```diff
@@ -47,3 +47,20 @@ VIDs can be generated via applications. Here are some tips:
 ## Define and modify the data type of VIDs
 
 The data type of VIDs must be defined when you [create the graph space](../3.ngql-guide/9.space-statements/1.create-space.md). Once defined, it cannot be modified.
+
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.vid.md:50]: pull and merge again

**Review by foesa-yang — DISMISSED** [review]

**Review by whitewum — COMMENTED** [review]
overrides/partials/header.html  don't delete this file

**Review by foesa-yang — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**